CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
     - Replaced read QC donut plots with a stacked bar chart (#168).
     - Replaced FastQC report links with MultiQC link (#169).
     - Revised report text, added figure captions and table titles (#172).
+    - Streamlined the Python code responsible for generating the report (#175).
```

Comment:
If we end up splitting these updates across multiple PRs, as I'd suggest, then we can just add all of the relevant PRs to this change log entry.